server/src/main/java/io/spine/server/EnvSetting.java

```diff
@@ -86,7 +85,6 @@
  * @param <V>
  *         the type of value
  */
-@Internal
```

Comment:
Not sure why this class is not internal anymore. Do we expect the framework users to use it directly?

Reply:
The class became `package-private` at one point, so I thought this annotation presence is just a typo.

Reply:
OK, now it makes sense.